stories/core/Table.stories.tsx

```diff
@@ -1676,3 +1682,140 @@ WithManualPaginator.argTypes = {
     docs: { source: { excludeDecorators: true } },
   },
 };
+
+export const ResizableColumns: Story<Partial<TableProps>> = (args) => {
+  type TableStoryDataType = {
+    index: number;
+    name: string;
+    description: string;
+    id: string;
+    startDate: Date;
+    endDate: Date;
+  };
+
+  const columns = useMemo(
+    (): Column<TableStoryDataType>[] => [
+      {
+        Header: 'Table',
+        columns: [
+          {
+            id: 'index',
+            Header: '#',
+            accessor: 'index',
+            width: 80,
+            disableResizing: true,
+          },
+          {
+            id: 'name',
+            Header: 'Name',
+            accessor: 'name',
+          },
+          {
+            id: 'description',
+            Header: 'Description',
+            accessor: 'description',
+            fieldType: 'text',
+            minWidth: 100,
+          },
+          {
+            id: 'id',
+            Header: 'ID',
+            accessor: 'id',
+            width: 100,
+            disableResizing: true,
+          },
+          {
+            id: 'startDate',
+            Header: 'Start date',
+            accessor: 'startDate',
+            Cell: (props: CellProps<TableStoryDataType>) => {
+              return props.row.original.startDate.toLocaleDateString('en-US');
+            },
+            width: 100,
+            disableResizing: true,
+          },
+          {
+            id: 'endDate',
+            Header: 'End date',
+            Cell: (props: CellProps<TableStoryDataType>) => {
+              return props.row.original.endDate.toLocaleDateString('en-US');
+            },
+            maxWidth: 200,
+          },
+        ],
+      },
+    ],
+    [],
+  );
+
+  const data = useMemo(
+    () => [
+      {
+        index: 1,
+        name: 'Name1',
+        description: 'Description1',
+        id: '111',
+        startDate: new Date('May 1, 2021'),
+        endDate: new Date('Jun 1, 2021'),
```

Comment:
If there is no min width maybe the text should be hidden and show "N..." instead of extending the cell vertically
![image](https://user-images.githubusercontent.com/88331924/144316137-a9afecb5-6943-492e-937b-9a385d9f1b94.png)

Reply:
yeah, i agree.. we need to set min width on columns...

Reply:
Also, different, but looks weird that I want to resize date, but description is resizing
![resize](https://user-images.githubusercontent.com/81580355/144583542-14a6db91-e7d1-4e7f-b3d1-6ce96b42f271.gif)

Reply:
so as we talked, that behavior might be ok, but need to handle last column

Reply:
Handling last column with disabled resizing.